### PR TITLE
You can now return to a brain. Also, clean up terrible console misuse.

### PR DESCRIPTION
**Pull request by 20kdc**
## About the PR

When ghosting from a brain, you weren't able to return to it - the link was severed. This PR allows doing so.
In addition, there is some misuse of `new Ghost().Execute(new ConsoleShell(host, session), string.Empty, Array.Empty<string>());` that is cleaned up.

Mind gets an explicit property to measure "physical deadness" for ghosting purposes, which corrects the issue with ghosting from brains.
This is separate to IC deadness so that changes to IC deadness won't allow arbitrary ghost-and-return for an otherwise 'actually alive' character. (This doesn't have any effect right now but is future-proofing.)

But basically, this is primarily preparation to ensure brains properly carry minds in case anyone wants to do something with that.

**Changelog**

:cl:
- fix: You no longer technically 'go catatonic' when ghosting from a brain or other 'effectively inanimate' object.

### CODE REVIEW

File: Content.Server/Commands/Chat/SuicideCommand.cs
Comment (ShadowCommander) [inline Content.Server/Commands/Chat/SuicideCommand.cs:129]: Discussed this with @Zumorica on Discord and it'd be better to replace calls to OnGhostAttempt with an EntityEvent if possible.

**Review by ShadowCommander — CHANGES_REQUESTED** [review]

File: Content.Server/Commands/Chat/SuicideCommand.cs
Comment (20kdc) [reply Content.Server/Commands/Chat/SuicideCommand.cs:129]: I think that's out of scope for this PR unless someone wants to give me an idea of where everything actually goes in this regard

Comment (ShadowCommander) [reply Content.Server/Commands/Chat/SuicideCommand.cs:129]: That's fine. Maybe just create an issue for that then.

**Review by ShadowCommander — APPROVED** [review]

File: Content.Server/Commands/Chat/SuicideCommand.cs
Comment (20kdc) [reply Content.Server/Commands/Chat/SuicideCommand.cs:129]: https://github.com/space-wizards/space-station-14/issues/4112

File: Content.Server/Commands/Observer/Ghost.cs
Comment (PJB3005) [inline Content.Server/Commands/Observer/Ghost.cs:19]: Well you can't really `Obsolete` this since this is still used by it being a command.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: Content.Server/Commands/Observer/Ghost.cs
Comment (20kdc) [reply Content.Server/Commands/Observer/Ghost.cs:19]: Only implicitly. I see the point, it's just... I really *really* don't want people to keep calling this directly

Comment (20kdc) [reply Content.Server/Commands/Observer/Ghost.cs]: Gotten rid of the attribute

**Comment by ShadowCommander** [comment]
When you finish all the changes in a review you should be able to click the changes requested drop down, then click the triple dots, and dismiss the review. That way it's easier for the other maintainers to see they've been addressed when looking at the PR list.

**Comment by 20kdc** [comment]
I cannot click the changes requested dropdown (no such dropdown exists) - keep in mind we have different levels of access.

**Review by ShadowCommander — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**
